.test/example-commands.sh

```diff
@@ -3,15 +3,49 @@
-# </build>
-# <push>
+SOURCE_DATE_EPOCH=1700741054 \
+	docker buildx build --progress=plain \
+	--provenance=mode=max \
+	--sbom=generator="$BASHBREW_BUILDKIT_SBOM_GENERATOR" \
+	--output '"type=oci","dest=temp.tar"' \
+	--annotation 'org.opencontainers.image.source=https://github.com/docker-library/docker.git#6d541d27b5dd12639e5a33a675ebca04d3837d74:24/cli' \
+	--annotation 'org.opencontainers.image.revision=6d541d27b5dd12639e5a33a675ebca04d3837d74' \
+	--annotation 'org.opencontainers.image.created=2023-11-23T12:04:14Z' \
+	--annotation 'org.opencontainers.image.version=24.0.7-cli' \
+	--annotation 'org.opencontainers.image.url=https://hub.docker.com/_/docker' \
+	--annotation 'manifest-descriptor:org.opencontainers.image.source=https://github.com/docker-library/docker.git#6d541d27b5dd12639e5a33a675ebca04d3837d74:24/cli' \
+	--annotation 'manifest-descriptor:org.opencontainers.image.revision=6d541d27b5dd12639e5a33a675ebca04d3837d74' \
+	--annotation 'manifest-descriptor:org.opencontainers.image.created=1970-01-01T00:00:00Z' \
+	--annotation 'manifest-descriptor:org.opencontainers.image.version=24.0.7-cli' \
+	--annotation 'manifest-descriptor:org.opencontainers.image.url=https://hub.docker.com/_/docker' \
+	--tag 'docker:24.0.7-cli' \
+	--tag 'docker:24.0-cli' \
+	--tag 'docker:24-cli' \
+	--tag 'docker:cli' \
+	--tag 'docker:24.0.7-cli-alpine3.18' \
+	--tag 'oisupport/staging-amd64:4b199ac326c74b3058a147e14f553af9e8e1659abc29bd3e82c9c9807b66ee43' \
+	--platform 'linux/amd64' \
+	--build-context 'alpine:3.18=docker-image://alpine:3.18@sha256:d695c3de6fcd8cfe3a6222b0358425d40adfd129a8a47c3416faff1a8aece389' \
+	--build-arg BUILDKIT_SYNTAX="$BASHBREW_BUILDKIT_SYNTAX" \
+	--file 'Dockerfile' \
+	'https://github.com/docker-library/docker.git#6d541d27b5dd12639e5a33a675ebca04d3837d74:24/cli'
 mkdir temp
 tar -xvf temp.tar -C temp
-jq '.manifests |= (del(.[].annotations) | unique)' temp/index.json > temp/index.json.new
+rm temp.tar
+jq '
+	.manifests |= (
+		del(.[].annotations)
```

Comment:
Why are the annotations deleted?

Reply:
https://github.com/docker-library/meta-scripts/pull/17/files#r1467971216 :+1:

Reply:
Then why do we add them in the build?

Reply:
So that they show up in the provenance data :sweat_smile: